ci_framework/roles/cert_manager/tasks/validate_certs.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create $HOME/bin dir
+  ansible.builtin.file:
+    path: "{{ lookup('env', 'HOME') }}/bin"
+    state: directory
+    mode: '0755'
+
+- name: Install cert-manager cmctl CLI
+  ansible.builtin.unarchive:
+    src:
+      "https://github.com/cert-manager/cert-manager/releases/latest/download/cmctl-linux-amd64.tar.gz"
```

Comment:
can we get a parameter for that one? In case we have to chose another version than `latest`